src/Microsoft.Health.Fhir.Core/Features/Operations/Export/CreateExportRequestHandler.cs

```diff
@@ -58,13 +67,15 @@ public async Task<CreateExportResponse> Handle(CreateExportRequest request, Canc
 
             string hash = JsonConvert.SerializeObject(hashObject).ComputeHash();
 
+            string storageAccountConnectionHash = Microsoft.Health.Core.Extensions.StringExtensions.ComputeHash(_exportJobConfiguration.StorageAccountConnection);
```

Comment:
Can we avoid computing the hash and set `storageAccountConnectionHash` to an empty string when `_exportJobConfiguration.StorageAccountConnection` is empty?